tensorlayer/layers/convolution.py

```diff
@@ -1560,79 +1548,81 @@ def deconv2d(prev_layer,
     b_init : initializer or None
         The initializer for the bias vector. If None, skip biases.
     W_init_args : dictionary
-        The arguments for the weight matrix initializer.
+        The arguments for the weight matrix initializer (For TF < 1.3).
     b_init_args : dictionary
-        The arguments for the bias vector initializer.
+        The arguments for the bias vector initializer (For TF < 1.3).
     name : str
         A unique layer name.
 
-    Returns
-    -------
-    :class:`Layer`
-        A :class:`DeConv2dLayer` object.
-
     """
 
-    logging.info("DeConv2d %s: n_filters:%s strides:%s pad:%s act:%s" % (name, str(n_filter), str(strides), padding, act.__name__))
+    @deprecated_alias(layer='prev_layer', end_support_version=1.9)  # TODO remove this line for the 1.9 release
```

Comment:
This class has a double deprecation, please change for
`@deprecated_alias(layer='prev_layer', n_out_channel='n_filter', end_support_version=1.9)`